css/pages/cart.css

```diff
@@ -0,0 +1,98 @@
+.cart-main {
+  max-width: 1000px;
+  margin: 1rem auto;
+}
+
+.cart-main .h1 {
+  margin: 2rem 1rem;
+}
+
+.cart-container {
+  margin-top: 2rem;
+  gap: 1rem;
+}
+
+.cart-products {
+  width: 50%;
+}
+
+.cart-products .icon {
+  font-size: 1.25rem;
+}
+
+.cart-container .card-image {
+  height: auto;
+}
+
+.product-count-block {
+  font-size: 1rem;
+}
+
+.product-count-block .count-action {
+  font-size: 1.5rem;
+  cursor: pointer;
+  border-radius: 50%;
+}
+
+.product-count-block .count {
+  border: 1px solid var(--clr-secondary-light);
+  padding: 0.1rem 0.75rem;
+}
+
+.cart-container .wishlist-cta {
+  font-size: 0.9rem;
+  cursor: pointer;
+}
+
+.cart-details {
+  width: 40%;
+}
+
+.coupons-block,
+.billing-block,
+.address-block {
+  border: 1px solid var(--clr-secondary-light);
+  padding: 0.5rem;
+}
+
+.coupons-container,
+.billing-container {
+  margin: 1rem 0 0.5rem 0;
+  font-size: 0.9rem;
+}
+
+.coupons-container .icon,
+.address-container .icon {
+  margin-right: 0.5rem;
+  font-size: 1rem;
+}
+
+.coupons-container .btn,
+.address-container .btn {
+  font-size: 0.75rem;
+  padding: 0.25rem 0.5rem;
+}
+
+.billing-container > div {
+  padding: 0.5rem;
+}
+
+.address-container {
+  font-size: 0.9rem;
```

Comment:
Utility classes for font sizes would be a good idea cause I see you are repeating `font-size: 0.9em/1rem;` in multiple places

Reply:
Thanks for pointing this out! Made the changes accordingly